### ISSUE TITLE
Showing progress bar when running `verdi storage maintain`.

### ISSUE THREAD

**Issue by unkcpz** [aiidateam/aiida-core#6538]
I am running the command on the repository that occupying >200GB. After checking with @giovannipizzi, he estimate it will cost ~8 h. I was panic not see anything is progress and my disk space keeps going lower.

**Comment by giovannipizzi** [comment]
I also discussed this with @khsrali @agoscinski @GeigerJ2 at some point, I habe the feeling there might even be some work already done in this direction? Just pinging them to avoid duplication of work. Note that this, I think, requires changes in disk objectstore, so should first be implemented there, and then called properly in AiiDA

**Comment by giovannipizzi** [comment]
Note also #5485

**Comment by khsrali** [comment]
yes, I opened a [PR](https://github.com/aiidateam/disk-objectstore/pull/171) in `disk-objectstore` sometime ago, 
@unkcpz would be great if you can review it.
